cmd/shuttle-proxy/main.go

```diff
@@ -104,19 +105,32 @@ func (p *Proxy) getEndpoints(c echo.Context) ([]string, error) {
 		return nil, c.String(code, err.Error())
 	}
 
-	if len(view.Settings.UploadEndpoints) == 0 {
-		return nil, fmt.Errorf("all upload endpoints are unavailable")
+	var endps []string
+	for _, endp := range view.Settings.UploadEndpoints {
```

Comment:
the values of `UploadEndpoints` has `/content/add` as a path, so we have to remove them